src/librustc_lint/lib.rs

```diff
@@ -289,6 +290,11 @@ pub fn register_builtins(store: &mut lint::LintStore, sess: Option<&Session>) {
             reference: "issue #50589 <https://github.com/rust-lang/rust/issues/50589>",
             edition: None,
         },
+        FutureIncompatibleInfo {
```

Comment:
I don't think this needs to be a future incompatible thing. I mean we have no plan to remove `...` patterns as of yet, even if we got noisier about deprecating them. I'd rather we not make people think we are changing the language more than we are =)